src/client/index.ts

```diff
@@ -68,6 +68,15 @@ export interface NotificationOptions {
      * Notification body text.
      *
      * This is the main notification content, displayed below the notification title. The notification will expand to fit the length of this text.
+     * Markdown is supported and **only** rendered when displaying a notification.
```

Comment:
Would try to put this in more user-facing language (feels like it's in the language of our specific markdown library), and I definitely had to stop and think about what it would mean for it to *not* only be rendered when displaying a notification.

"Markdown may be used in the body text to control how it is styled when rendered. With the exception of links and images, all basic syntax as documented \[here\](https://www.markdownguide.org/basic-syntax) is supported."?

(Is code rendering support a deliberate omission?)

Reply:
Is being omitted for now.

Updated with your description + code not allowed.